Scripts/Sources/Automation/Eval.swift

```diff
@@ -0,0 +1,329 @@
+        
+        if isSpecificJob() {
+            return
+        }
+        
+        if TravisCI.isPullRquestJob() || Shell.nextArg("--env") == "pr" {
+            runPullRequestLane()
+        } else {
+            runContinousIntegrationLane()
+        }
+    }
+    
+    static func runPullRequestLane() {
+        runCommands("Building Pull Request") {
+            try prepareForBuild()
+            try build()
+            try runTests()
+            
+            try prepareExamplesForBuild()
+            try buildExamples()
+            try runTestsOnExamples()
+            
+            try runLinter()
+            try runDanger()
+        }
+    }
+    
+    static func runContinousIntegrationLane() {
+        runCommands("Building CI") {
+            try prepareForBuild()
+            try build()
+            try runTests()
+            
+            try prepareExamplesForBuild()
+            try buildExamples()
+            try runTestsOnExamples()
+            
+            try generateDocs()
+            try publishDocs()
+            
+            try runLinter()
+            try runCocoaPodsLinter()
+            
+            try testCoverage()
+            
+            try runDanger()
+        }
+    }
+    
+    static func isSpecificJob() -> Bool {
+        guard let jobsString = Shell.nextArg("--jobs") else { return false }
+        let jobsToRun = jobsString.split(separator: ",").map { String($0) }
+        let jobsFound = jobsToRun.flatMap { job in jobs.first { $0.key == job } }
+        runCommands("Executing jobs: \(jobsString)") {
+            if let job = jobsToRun.first(where: { !self.jobs.keys.contains($0) }) {
+                throw CIError.logicalError(message: "Job not found: \(job)")
+            }
+            try jobsFound.forEach {
+                print("🏃🏻 Running job \($0.key)")
+                try $0.value()
+            }
+        }
+        return !jobsFound.isEmpty
+    }
+    
+    static func runCommands(_ title: String, commands: () throws -> Void) {
+        do {
+            if !TravisCI.isRunningLocally() {
+                print("travis_fold:start: \(title)")
+            }
+            
+            print("ℹ️ \(title)")
+            try commands()
+            
+            if !TravisCI.isRunningLocally() {
+                print("travis_fold:end: \(title)")
+            }
+            
+            print("🎉 Finished successfully")
+        } catch let CIError.invalidExitCode(statusCode, errorOutput) {
+            print("😢 Error happened: [InsufficientExitCode] ", errorOutput ?? "unknown error")
+            exit(statusCode)
+        } catch let CIError.logicalError(message) {
+            print("😢 Error happened: [LogicalError] ", message)
+            exit(-1)
+        } catch CIError.timeout {
+            print("🕙 Timeout")
+            exit(-1)
+        } catch {
+            print("😢 Error happened [General]")
+            exit(-1)
+        }
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tevelee/Eval/blob/0e45c7188f4e60e9487408f3d9b69e8a8f17f368/Scripts/Sources/Automation/Eval.swift#L101"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>